requirements.txt

```diff
@@ -57,4 +57,6 @@ python-dotenv~=0.17.0
 setuptools~=65.5
 deprecated~=1.2
 jinja2~=3.1
-anyio~=3.5
+astor~=0.8.1
+databricks-sdk~=0.1.8
```

Comment:
There are already newer versions – 0.3.0, and 0.1.12 – https://pypi.org/project/databricks-sdk/#history

Should we open this up for later versions as well? Anything special about 0.1.8?

Reply:
Also, should we consider making it an extra requirement?

Reply:
let's stay with this version for now, Tomer please add a note to yourself to address both (move to extra-requirments and to latest version) in a following PR.